js/flickity.js

```diff
@@ -752,6 +752,27 @@ proto.queryCell = function( selector ) {
   return this.getCell( selector );
 };
 
+proto.checkVisibility = function() {
+  var viewportX = this.viewport.getBoundingClientRect().x;
+  var viewportWidth = this.viewport.offsetWidth;
+
+  this.cells.forEach(function (cell) {
+    var cellX = cell.element.getBoundingClientRect().x - viewportX;
+    var isVisible = (
```

Comment:
`isVisible` will not function correctly unless the slider is full width. Anything in-line will not work if the cell is not wider than the viewport offset. 

I have added a test to check if the value of `cellX === 0` as it is the first element within the viewport if the value of cellX is 0 (More specifically I am checking if `cellX > -1 && cellX < 1` due to the fluctuation of cell placement when the carousel stops moving).

This is not an ideal solution as I am getting flashing before a slide comes to rest, but it is solving the issue in my use case.